### ISSUE TITLE
liveimage-mount dies with SyntaxError

### ISSUE THREAD

**Issue by jflorian** [livecd-tools/livecd-tools#73]
```
$  liveimage-mount
  File "/usr/bin/liveimage-mount", line 715
    environ['PS1'] = X + '-[\u@\h \W]\$ '
                        ^
SyntaxError: (unicode error) 'unicodeescape' codec can't decode bytes in position 2-3: truncated \uXXXX escape

$  rpm -q livecd-tools
livecd-tools-24.4-1.fc26.x86_64
```

**Comment by FGrose** [comment]
This should be fixed with the latest commit in pull request #66.

**Comment by Conan-Kudo** [comment]
This is fixed with 916bf57eb356064dabd184b5222ecd8fb4346595.
